### PR TITLE
JOSS paper

### PR DESCRIPTION
**Pull request by streeve**
Creating a PR mostly to keep this in the front of our minds. We can merge or keep as a separate branch when we're done

If anyone has ideas, let's discuss and get it submitted soon. PDF is compiled under "Actions": `.github/workflows/draft-pdf.yml`

### CODE REVIEW

**Review by sslattery — APPROVED** [review]
I have added my edits/additions. If we want a code sample for AoS vs. SoA I have added a note to what this might look like. Not sure it is necessary though.

**Comment by streeve** [comment]
@YuxingQiu and @aetx can you check the brief descriptions of the sparse grid / load balancing, respectively?

**Comment by streeve** [comment]
Anyone have anything to add or fix? Likely try to submit soon

File: paper/paper.bib [hunk 1]
```diff
@@ -184,3 +184,10 @@ @inproceedings{belak2019exaam
   pages={C22--010},
   year={2019}
 }
+
+@misc{all,
+    Author = {R.~Halver and S.~Schulz and G.~Sutmann},
+    Howpublished = {{\verb+https://gitlab.version.fz-juelich.de/SLMS/loadbalancing/-/releases+}},
+    Title = {{ALL - A loadbalancing library, C++ / Fortran library}},
+    Url = {https://gitlab.version.fz-juelich.de/SLMS/loadbalancing/-/releases},
```
Comment (aetx) [inline paper/paper.bib:192]: We would rather link to the webpage itself (http://slms.pages.jsc.fz-juelich.de/websites/all-website/) than the GitLab one. Maybe just change URL to the webpage and leave howpublished alone to link to both.

Comment (streeve) [reply paper/paper.bib]: Done

**Review by aetx — APPROVED** [review]

**Pull request closed without merging**